apps/detection/src/providers/sentiment-analysis/sentiment-analysis.chatgpt.service.ts

```diff
@@ -31,7 +31,7 @@ Never generate an answer, by default anser with ${defaultEmotion}|1.0`;
     try {
       content = await this.llmProvider.chat({
         system,
-        message,
+        user: message,
```

Comment:
I guess it's just the llmProvider.chat() method signature, but it puzzles me a bit the fact that a user is a message.

Reply:
yes indeed. The chat accepts messages: LLMMessageDto[] which often is composed of a system message + user message. As a short hand, the API accept a user + message params. API consumer around the code adapts somehow. Do you have a suggestion to improve that snippet ?